templates/documentation/indicators.html

```diff
@@ -14,7 +14,67 @@ <h2>Climate Indicators</h2>
 
 <h3>Service endpoints</h3>
 
-<h4>All climate indicator variables</h4>
+<h4>CMIP6 climate indicator variables</h4>
+
+<p>
+  Query CMIP6 climate indicator variables for all models and scenarios.
+</p>
+
```

Comment:
I suggest adding in here what the indicators themselves actually are / mean.  Like a simple list with the definitions of the indicators.

Reply:
NM it's at the bottom!  But how about here, add a link (anchor to bottom) where it says just "See below for definitions of indicators included here."